includes/class-woothemes-sensei-utils.php

```diff
@@ -898,6 +898,7 @@ public static function sensei_get_quiz_questions( $quiz_id = 0 ) {
 	}
 
 	public static function sensei_get_quiz_total( $quiz_id = 0 ) {
+		global $woothemes_sensei;
```

Comment:
@davidpagepsycle Thank you for this. 

Just a note: we've introduced Sensei() global function which you can use like this:

Sensei()->question->function_name

We will be retiring the global object in future.